### PR TITLE
Thomasnb/bug/cleaning up comments and adding more accurate names

### PR DESCRIPTION
**Pull request by noahthomas88**

### CODE REVIEW

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
 import { UserlistComponent } from './userlist/userlist.component';
 import { DashboardComponent } from './dashboard/dashboard.component';
-import { CaseComponent } from './case/case.component';
+import { PositiveCaseComponent } from './positiveCase/positiveCase.component';
```
Comment (meistejw) [inline src/app/app-routing.module.ts:5]: Why is changing the name of a component so hard in Angular???

File: src/app/person-profile/person-profile.component.html [hunk 2]
```diff
@@ -1,42 +1,35 @@
 <table mat-table [dataSource]="dataSource" class="mat-elevation-z8">
 
-  <!-- Name Column -->
```
Comment (meistejw) [inline src/app/person-profile/person-profile.component.html:3]: Goodbye to all the useless Name Column placeholders from copy and pasting!

File: src/app/person-profile/person-profile.component.html [hunk 3]
```diff
@@ -51,4 +44,4 @@
 
   <mat-header-row *matHeaderRowDef="displayedColumns"></mat-header-row>
   <mat-row *matRowDef="let row; columns: displayedColumns;"></mat-row>
-</table>
```
Comment (meistejw) [inline src/app/person-profile/person-profile.component.html:54]: What was the issue here exactly?

**Review by meistejw — CHANGES_REQUESTED** [review]
This code does not actually compile.

There appear to be more issues with renaming the case component.

File: src/app/person-profile/person-profile.component.html (hunk 2 above)
Comment (noahthomas88) [reply src/app/person-profile/person-profile.component.html:3]: yay

File: src/app/positiveCase/positiveCase.service.ts [hunk 4]
```diff
@@ -6,7 +6,7 @@ import { catchError, retry } from 'rxjs/operators';
 @Injectable({
   providedIn: 'root'
 })
-export class CaseService {
+export class PositiveCaseService {
```
Comment (meistejw) [inline src/app/positiveCase/positiveCase.service.ts:9]: This appears to have fixed the errors.

**Review by meistejw — APPROVED** [review]
Looks good. The code compiles and removes a lot of unnecessary stuff from our code.

**Pull request merged**